### PR TITLE
ENS send flow e2e

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes TEAM2-11

## PoW (screenshots / screen recordings)

E2E are failing since there are still things to fix in the main branch

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels?
- [ ] Added e2e tests? if not please specify why
- [ ] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

File: src/helpers/ens.ts [hunk 1]
```diff
@@ -419,9 +419,13 @@ const getENSBaseRegistrarImplementationContract = async (wallet?: Wallet) => {
   );
 };
 
-const getENSRegistryContract = async () => {
-  const provider = await getProviderForNetwork();
-  return new Contract(ensRegistryAddress, ENSRegistryWithFallbackABI, provider);
+const getENSRegistryContract = async (wallet?: Wallet) => {
+  const signerOrProvider = wallet || (await getProviderForNetwork());
```
Comment (estebanmino) [inline src/helpers/ens.ts:423]: using wallet to be able to send the `setOwner` tx

File: src/helpers/ens.ts [hunk 2]
```diff
@@ -507,7 +511,9 @@ const setupMulticallRecords = (
   if (textAssociatedRecord) {
     data.push(
       textAssociatedRecord
-        .filter(textRecord => Boolean(textRecord.value))
+        .filter(
+          textRecord => Boolean(textRecord.value) || textRecord.value === ''
```
Comment (estebanmino) [inline src/helpers/ens.ts:515]: i feel like this one was also affecting users that erased a record manually in the ens assign records form
this is filtering out those empty string records

Comment (jxom) [reply src/helpers/ens.ts:515]: good catch

**Review by jxom — APPROVED** [review]
looks good! I'll merge and have a look at the failing test

**Pull request merged**